app/javascript/components/App.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+// import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Greeting from './Greeting';
+
+const App = () => {
+  return (
+    <Greeting />
+  );
+};
```

Comment:
- [ ] Please also add the `react-router` library and adjust your code accordingly, so that a user should be able to see `random` greetings on the home page `redirected` via `react-router`.

![image](https://github.com/Omwomo/hello-rails-react/assets/66476329/70cb9108-53c3-4b84-abc0-b0cc37df15a7)